cmd/livepeer/starter/starter.go

```diff
@@ -1291,9 +1291,17 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 			}
 			if *cfg.AIWorker {
 				modelConstraint := &core.ModelConstraint{Warm: config.Warm, Capacity: 1}
-				// External containers do auto-scale; default to 1 or use provided capacity.
-				if config.URL != "" && config.Capacity != 0 {
-					modelConstraint.Capacity = config.Capacity
+				modelsCount := 1
+				if config.Capacity != 0 {
+					if config.URL == "" {
+						// Use multiple same configs if External Container is not used and capacity is set
+						modelsCount = config.Capacity
+					} else {
+						// External containers do auto-scale; default to 1 or use provided capacity.
+						if config.URL != "" && config.Capacity != 0 {
```

Comment:
Seems like this check is redundant at this point? Both should always be true. Also, the comment seems inaccurate now. So maybe we don't even need to check `config.URL` here and just have `modelsCount = config.Capacity`?

Reply:
Oh nevermind I saw that we set a different thing when it's external. So maybe only drop the inner `if`

Reply:
Good idea, updated.